tests/fsm/test_json_schema.py

```diff
@@ -4,966 +4,8 @@
 
 import interegular
 import pytest
-from outlines_core.fsm.json_schema import (
-    BOOLEAN,
-    DATE,
-    DATE_TIME,
-    INTEGER,
-    NULL,
-    NUMBER,
-    STRING,
-    STRING_INNER,
-    TIME,
-    UUID,
-    WHITESPACE,
-    build_regex_from_schema,
-    to_regex,
-)
-from pydantic import BaseModel, Field, constr
-
-
-def test_from_pydantic():
```

Comment:
Let's keep this one too

Reply:
Ah, noticed the comment that it'll be in `outlines`, all good then 👍 : https://github.com/dottxt-ai/outlines-core/issues/117#issuecomment-2527791781